docs/commands.md

```diff
@@ -127,11 +130,11 @@ The `add-mask`, `debug`, `warning` and `error` commands do not support echoing.
 ### Command Prompt
 
 CMD processes the `"` character differently from other shells when echoing. In CMD, the above snippets should have the `"` characters removed in order to correctly process. For example, the set output command would be:
+
 ```cmd
 echo ::set-output name=FOO::BAR
 ```
 
-
 # Environment files
```

Comment:
This should actually be a h2 shouldn't it? But wasn't sure if it's consumed by some script which requires a h1 here...

```suggestion
## Environment files
```